charts/internal/gvisor/templates/rbac-gvisor.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gvisor
```

Comment:
Multiple gvisor installations for different worker pools will all reconciles/overwrite these objects, right?

Reply:
see above.